jre6.txt

```diff
@@ -0,0 +1,41 @@
+[Section]
+Name = Java SE Runtime Environment 6
+Version = 6u45
+License = Oracle Binary Code License
+Description = Platform and libraries needed for running Java-based programs.
+Size = 16.2 MiB
+Category = 14
+URLDownload = http://svn.reactos.org/packages/jre-6u45-windows-i586.exe
```

Comment:
I'm not sure Oracle permits redistribution of these packages.

Reply:
You already have a JRE 7u80 installer, so I assumed that you have an agreement/it's allowed to redistribute these packages.